RadixWallet/EngineKit/IsXRDResource.swift

```diff
@@ -15,6 +15,12 @@ extension ResourceAddress {
 	static let mainnetXRDAddress: ResourceAddress = .xrd(on: .mainnet)
 }
 
+extension SpecificAddress {
+	var isOnMainnet: Bool {
+		networkID == NetworkID.mainnet
```

Comment:
Please out put this in another file. Please :D `RadixWallet/EngineKit/` is verboten! :D

Reply:
ohh yes, forgoten it is verboten :D